lib/thumbyGrayscale.py

```diff
@@ -1123,7 +1123,7 @@ def blitWithMask(self, src, x:int, y:int, width:int, height:int, key:int, mirror
             dstco = dsto
             i = width
             while i != 0:
-                if (maskp[srcco] & srcm) == 0:
+                if maskp[srcco] & srcm:
```

Comment:
That's super weird that we got that the wrong way around..?! Do you think the Thumby API has swapped this in the mean time..?

Reply:
I actually think we might have had it the wrong way around from the beginning! But I checked a masked sprite from one of the games on the Arcade and it's indeed opposite!

We will just need to note this is technically a breaking change from v3.0.0b, and RocketCup is using masked sprites, so I'm recommending a v4.0.0 release in good time.